packages/task/src/node/task-server.slow-spec.ts

```diff
@@ -26,26 +26,28 @@ import { terminalsPath } from '@theia/terminal/lib/common/terminal-protocol';
 import { expectThrowsAsync } from '@theia/core/lib/common/test/expect';
 import { TestWebSocketChannel } from '@theia/core/lib/node/messaging/test/test-web-socket-channel';
 import { expect } from 'chai';
+import URI from '@theia/core/lib/common/uri';
 
 /**
  * Globals
  */
 
 // test scripts that we bundle with tasks
 const commandShortRunning = './task';
-const commandShortrunningindows = 'task.bat';
+const commandShortrunningindows = '.\\task.bat';
 
 const commandLongRunning = './task-long-running';
-const commandLongRunningWindows = 'task-long-running.bat';
+const commandLongRunningWindows = '.\\task-long-running.bat';
 
 const bogusCommand = 'thisisnotavalidcommand';
 
-const commandToFindInPathUnix = 'ls';
-const commandToFindInPathWindows = 'dir';
```

Comment:
Turns out that `dir` is not a real executable, but only exist in a `cmd.exe` prompt...
https://superuser.com/a/229949